olm/0.4.0/kong.v0.4.0.clusterserviceversion.yaml

```
@@ -6,29 +6,45 @@ metadata:
+    createdAt: '2020-08-14T11:43:45Z'
+    description: The world’s most popular open source API gateway. Built for multi-cloud and hybrid, optimized for microservices and distributed architectures.
+    olm.skipRanges: '>=0.2.6 <0.4.0'
     repository: https://github.com/kong/kong-operator
     support: Harry Bagdi
-  name: kong.v0.3.0
+  name: kong.v0.4.0
   namespace: placeholder
 spec:
   customresourcedefinitions:
     owned:
-    - description: A Kong cluster configuration. Mirrors the settings in the Helm chart's values.yaml
+    - description: Defines a Kong cluster (equivalent to a Helm release).
       displayName: Kong
       kind: Kong
       name: kongs.charts.helm.k8s.io
       version: v1alpha1
-  description: >
-    Kong is a popular open-source cloud-native API gateway.
+  description: |
+    Kong is a popular open-source cloud-native API gateway. Kong Operator is a Kubernetes operator which manages [Kong](https://konghq.com/kong/) and [Kong Enterprise](https://konghq.com/products/kong-enterprise/) clusters.
 
-    The Kong Operator manages [Kong](https://konghq.com/kong/) and [Kong Enterprise](https://konghq.com/products/kong-enterprise/) clusters.
-    It includes an [ingress controller](https://github.com/Kong/kubernetes-ingress-controller) to configure Kong's gateway using Kubernetes resources.
+    Kong Operator can deploy Kong in various configurations, for example:
+    * as a [Kubernetes ingress controller](https://github.com/Kong/kubernetes-ingress-controller), enabling you to expose Kubernetes `Service`s via Kong,
+    * a standalone Kong gateway (without the ingress controller; either DB-enabled or DB-less)
+    * a standalone Ingress Controller (configuring an external instance of Kong)
 
-    The operator manages Kong using the [Kong Helm chart](https://github.com/Kong/charts/blob/master/charts/kong/README.md).
-    Its `Kong` custom resource contains a [values.yaml](https://github.com/Kong/charts/blob/master/charts/kong/values.yaml).
+    Those configurations are further described in the [_Deployment Options_](https://github.com/Kong/charts/blob/v0.4.0/charts/kong/README.md#deployment-options) section of documentation.
+
+    ### Quick Start
+
+    The [Quick Start guide](https://github.com/Kong/kong-operator/blob/v0.4.0/README.md#quick-start) uses Kong Operator to deploy
+    our recommended Kong setup for Kubernetes users (Kong DB-less with Ingress Controller), which includes an instance of Kong
+    serving as a proxy to an example Kubernetes service.
+
+    ### Documentation
+
+    The primary documentation site for Kong Operator is located [in its GitHub repository](https://github.com/Kong/kong-operator/blob/v0.4.0/README.md).
+
+    Kong Operator provides the same configuration flexibility as the Kong Helm chart.
+    Refer to the [documentation of the Helm chart](https://github.com/Kong/kong-operator/blob/v0.4.0/helm-charts/kong/README.md) used by this version of Kong Operator,
```

@hbagdi IIRC for previous releases we decided to keep links to the Helm docs version-agnostic, to reduce what we need to update in each version, even though OLM files are versioned?

Link change to the Helm README (it looks like the original was a clipboard mistake), some prose adjustments:
```suggestion
    Refer to the [Helm chart documentation](https://github.com/Kong/charts/blob/master/charts/kong/README.md),
```
If we do instead want to use the version link, https://github.com/Kong/charts/blob/kong-1.8.0/charts/kong/README.md

---

The original is not a clipboard mistake - it directly links to the helm chart used by specifically this version of operator.

IMO it makes sense to associate a Helm chart version with operator version because configuration options differ from version to version, and there'll be a lot of confusion if one tries to use an option not available in their version of the chart.
If we apply this suggestion, we'll need to clarify the mapping between the helm chart version and the operator version somewhere else.

My preference is to keep the original, but I'm open to discussion.